### PR TITLE
Drupal core 8.5.x updates

### PR DESCRIPTION
**Pull request by kolesnikoff**
@podarok this PR is mostly about how we can update Migrate modules.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy.cibox.tools/build304 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build304 |
| Installation process | http://install.openy-dev.ffwua.com/build304/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/304/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build304/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build304/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build304/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build304/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build304/blogWCAG2AAhtmlcs.txt

File: modules/custom/openy_socrates/src/OpenySocratesFacade.php [hunk 1]
```diff
@@ -82,33 +82,33 @@ public function __call($name, array $arguments) {
...
    * Setter for services tagged with 'openy_cron_service' tag.
    *
    * @param array $services
    *   Services.
    */
-  public function collectCronServices(array $services) {
+  public function collectCronServices($service, $priority, $periodicity) {
     /** @var OpenyCronServiceInterface $service */
-    foreach ($services as $periodicity => $service) {
+//    foreach ($services as $periodicity => $service) {
```
Comment (danylevskyi) [inline modules/custom/openy_socrates/src/OpenySocratesFacade.php:109]: Is this finished PR?
I think we should avoid commenting the old code.

**Review by danylevskyi — CHANGES_REQUESTED** [review]
Need's removing comments with old code.

File: modules/custom/openy_socrates/src/OpenySocratesFacade.php (hunk 1 above)
Comment (kolesnikoff) [reply modules/custom/openy_socrates/src/OpenySocratesFacade.php:109]: @danylevskyi , this PR is just a possible solution to update Migrate module to the latest version.
All other changes are taken from one of the previous PRs, where Andrii has tried to update Drupal to v8.5.

**Comment by ddrozdik** [comment]
Closing this, since we will upgrade to Drupal 8.6 in this PR https://github.com/ymcatwincities/openy/pull/1395

**Pull request closed without merging**